kinect_bridge2/KinectSpeechPhrase[] phrases
